versions/1/_dev/deploy/tf/spec.yml

```diff
@@ -5,9 +5,22 @@ spec:
       pattern: '^*.tf$'
       type: file
       required: true
+    - description: Terraform dependency lock file
+      pattern: ".terraform.lock.hcl"
+      type: file
+      required: true
+    - description: Terraform template files
+      pattern: '^*.tftpl$'
+      type: file
+      required: false
     - description: Environment variables exposed for Terraform
       type: file
       name: "env.yml"
       contentMediaType: "application/x-yaml"
       required: false
       $ref: "./env.spec.yml"
+    - description: Fixture JSON files
+      pattern: '^*.json$'
+      type: file
+      contentMediaType: "application/json"
+      required: false
```

Comment:
Perhaps we end-up needing `additionalContents: false` if we find scenarios that would need arbitrary files. But we can reduce restrictions in the future if we find these cases.